### PR TITLE
Feature: Silent post init hooks

### PR DESCRIPTION
**Pull request by doosuu**
Implements #58 . Post init hooks now are silent by default. Full logging can be enabled with the verbose flag.

### CODE REVIEW

File: src/commands/init/index.ts [hunk 1]
```diff
@@ -13,6 +13,7 @@
 // SPDX-License-Identifier: Apache-2.0
 
 import { Command, Flags } from '@oclif/core';
+import { ux } from '@oclif/core/lib/cli-ux';
```
Comment (dennismeister93) [inline src/commands/init/index.ts:16]: could you adapt either this or the import of "CliUx" in commands/upgrade/index.ts to be consistent?

File: src/modules/exec.ts [hunk 2]
```diff
@@ -76,7 +85,8 @@ export async function runExecSpec(
     componentId: string,
     projectConfig: ProjectConfig,
     envVars: NodeJS.ProcessEnv,
-    verbose?: boolean
+    writeStdout: boolean,
```
Comment (dennismeister93) [inline src/modules/exec.ts:88]: what do you think of having an object here? For better understanding when setting this?
sth like stdOutOptions: {verboseLog:boolean, writeStdOut:boolean}

File: src/commands/init/index.ts (hunk 1 above)
Comment (doosuu) [reply src/commands/init/index.ts:16]: Adapted it here, auto import made a mess of this.

**Review by dennismeister93 — APPROVED** [review]
lgtm

**Pull request merged**